app/destination/page.js

```diff
@@ -1,54 +1,102 @@
 "use client";
 
-import { useState } from 'react';
+import { useState } from "react";
 
-import styles from '@/components/destination/destination.module.css';
-import { AddWishlistItem } from '@/components/destination/AddWishlistItem';
+import styles from "@/components/destination/destination.module.css";
+import { AddWishlistItem } from "@/components/destination/AddWishlistItem";
 
-// TASK - React 1 week 2
-// Move this to its own file
-const PlanetWishlistItem = ({
+const PlanetCard = ({
   name,
-  onRemove,
+  description,
   thumbnail,
+  isSelected,
+  index,
+  onAddOrRemovePlanet,
```

Comment:
```suggestion
  onClick,
```
Prefer using a generic name for interaction event handlers. It will make your components more reusable.